### DISCUSSION TITLE
[Query] How to label Dependabot Security Alert PRs

### DISCUSSION THREAD

**Discussion by ejntaylor** [crazy-max/ghaction-github-labeler#194] [Q&A] [answered]
Dependabot creates two types of pull requests - standard version bumps and security vulnerability bumps.

Is there a way we can use your labeler action to add a specific 'security' label to the latter?

Thanks

**Comment by crazy-max** [comment, marked as answer]
@raisonon Not possible and not the purpose of this action. I think it has to be done on Dependabot side with a new attribute like `security-labels`. Suggest to leave an issue on their repo: https://github.com/dependabot/dependabot-core

**Comment by ejntaylor** [comment]
Thanks. For reference: https://github.com/dependabot/dependabot-core/issues/3726
